Cargo.toml

```diff
@@ -21,16 +21,20 @@ exclude       = [
 lazycell = "0.4.0"
 log      = "0.3.1"
 slab     = "0.3.0"
-net2     = "0.2.19"
```

Comment:
`net2` is not ported to Redox yet. It has been added to the `cfg(unix)` and `cfg(windows)` dependencies.